app/views/shared/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <% environment_style = GovukAdminTemplate.environment_style %>
 <% environment_label = GovukAdminTemplate.environment_label %>
 <% environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment %>
-<% show_notices ||= true %>
+<% show_legacy_notices ||= false %>
```

Comment:
It seems like defining defaults in this way is just inviting these kind of bugs. Do you know if ruby/rails offer any better alternatives?

Reply:
This is currently our convention on setting defaults, see other examples: https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_cards.html.erb#L2

I do get what you're saying, however, this partial is only used by two templates, admin layout and design system layout.

Reply:
I am going to merge, but feel free to suggest any alternative solutions.